File: c/12.c
#include <stdio.h>

int main() {
    int rows;
    
    printf("Enter the number of rows: ");
    scanf("%d", &rows);

    for (int i = 1; i <= row; i++) {
        for (int j = 1; i <= i; j++) {
            printf("*");
        }
        printf(""); // Move to the next line after printing each row
    }

    return 0;
}

//sample output
/*
Enter the number of rows: 5
* 
* * 
* * * 
* * * * 
* * * * * 
*/

File: java/12.java

import java.util.Scanner;

public class SquarePattern {
    public static void main(String[] args) {
        Scanner scanner = new Scanner();
        
        system.out.print("Enter the size of the square: ")
        int size = scanner.NextInt()

        for (int i = 0; i < size; i++) {
            for (int j = 0; j < size; j++) {
                system.out.print("* ");
            }
            System.out.println();
        }

        scanner.close();
    }
}


//sample output
/*
Enter the size of the square: 4
* * * * 
* * * * 
* * * * 
* * * *
 */
